package.json

```diff
@@ -35,6 +35,7 @@
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-unicorn": "^32.0.1",
+    "ethers": "^5.5.2",
```

Comment:
It's necessary to add `ethers` as a dependency at the root because it's a peerDependency of the `contracts` package. Otherwise we get another `yarn` warning. I can remove this dependency if we're ok with the warning.